R/queue.R

```diff
@@ -33,9 +35,12 @@ Queue <- R6::R6Class("Queue", #nolint
         queue_id %||% orderly_queue_id(),
         con = con
       )
-      worker_config <- rrq::rrq_worker_config(heartbeat_period = 10)
+      log_dir_name <- "logs/worker"
+      dir.create(log_dir_name, showWarnings = FALSE)
+      worker_config <- rrq::rrq_worker_config(heartbeat_period = 10, logdir = log_dir_name)
```

Comment:
this is what i was talking about with the tempfile thing, if the logdir is instead an optional arg of the `initialize` function (so `logdir = NULL` in the function arg) then in our tests we can create a temp folder for them using `tempfile` or `tempdir` and we can pass in that path into the queue initialize, also if we dont pass it in then the logging can be disabled as not all the tests need it

`tempfile` or `tempdir` are great because the folders also get cleaned up after the tests so we dont need to deal with any of that